inference-engine/src/vpu/graph_transformer/src/middleend/passes/replace_with_reduce_mean.cpp

```diff
@@ -37,38 +37,38 @@ void PassImpl::run(const Model& model) {
         }
 
         auto stageInput = stage->input(0);
-        const auto& dimValues = stageInput->desc().dims();
+        auto stageOutput = stage->output(0);
```

Comment:
Why by non-const value?

Reply:
Did you just moved that code? If so, it looks like a nitpick, but anyway..

Reply:
Fixed